addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/QuickStartParam.java

```diff
@@ -0,0 +1,215 @@
+        }
+        try {
+            recentUrls = getConfig().getList(PARAM_RECENT_URLS, new ArrayList<Object>(0));
+        } catch (Exception e) {
+            LOGGER.error("Failed to load the recent urls configuration", e);
+        }
+        try {
+            maxRecentUrls = getConfig().getInt(PARAM_MAX_RECENT_URLS, 5);
+        } catch (Exception e) {
+            LOGGER.error("Failed to load the recent urls configuration", e);
+        }
+        try {
+            launchStartPage = getConfig().getString(PARAM_START_PAGE, ZAP_START_PAGE);
+        } catch (Exception e) {
+            LOGGER.error("Failed to load the \"Start Page\" configuration", e);
+        }
+        try {
+            launchDefaultBrowser = getConfig().getString(PARAM_DEFAULT_BROWSER, DEFAULT_BROWSER);
+        } catch (Exception e) {
+            LOGGER.error("Failed to load the \"Default Browser\" configuration", e);
+        }
+        try {
+            isAjaxSpiderEnabled = getConfig().getBoolean(PARAM_AJAX_SPIDER_ENABLED, false);
+        } catch (Exception e) {
+            LOGGER.error("Failed to load the ajax spider configuration", e);
+        }
+        try {
+            ajaxSpiderDefaultBrowser = getConfig().getString(PARAM_AJAX_SPIDER_DEFAULT_BROWSER, DEFAULT_BROWSER);
+        } catch (Exception e) {
+            LOGGER.error("Failed to load the Ajax \"Default Browser\" configuration", e);
+        }
+    }
+
+    public String getLaunchStartPage() {
+        return launchStartPage;
+    }
+
+    public boolean isLaunchZapStartPage() {
+        return ZAP_START_PAGE.equals(launchStartPage);
+    }
+
+    public boolean isLaunchBlankStartPage() {
+        return BLANK_START_PAGE.equals(launchStartPage);
+    }
+
+    public void setLaunchZapStartPage() {
+        setLaunchStartPage(ZAP_START_PAGE);
+    }
+
+    public void setLaunchBlankStartPage() {
+        setLaunchStartPage(BLANK_START_PAGE);
+    }
+
+    public void setLaunchStartPage(URL url) {
+        if (url == null) {
+            setLaunchZapStartPage();
+        } else {
+            setLaunchStartPage(url.toString());
+        }
+    }
+
+    private void setLaunchStartPage(String str) {
+        this.launchStartPage = str;
+        getConfig().setProperty(PARAM_START_PAGE, str);
+    }
+
+    public String getLaunchDefaultBrowser() {
+        return launchDefaultBrowser;
+    }
+
+    public void setLaunchDefaultBrowser(String defaultBrowser) {
+        this.launchDefaultBrowser = defaultBrowser;
+        getConfig().setProperty(PARAM_DEFAULT_BROWSER, defaultBrowser);
+    }
+
+    public boolean isTradSpiderEnabled() {
+        return this.isTradSpiderEnabled;
+    }
+
+    public void setTradSpiderEnabled(boolean isTradSpiderEnabled) {
+        this.isTradSpiderEnabled = isTradSpiderEnabled;
+        getConfig().setProperty(PARAM_TRAD_SPIDER_ENABLED, isTradSpiderEnabled);
+        try {
+            getConfig().save();
```

Comment:
Does it need to save? We don't usually do that in other params (ZAP already saves during shutdown).